environment.yml

```diff
@@ -16,60 +16,48 @@ dependencies:
 - cligj=0.4.0=py27_0
 - curl>=7.49.1
 - cycler=0.10.0=py27_0
-- descartes=1.1.0=py27_0
 - enum34=1.1.6=py27_1
 - ephem=3.7.6.0=py27_1
 - expat=2.2.5
-- fiona=1.7.11
 - freetype=2.7=vc9_1
 - freexl=1.0.2=vc9_2
 - functools32=3.2.3.2=py27_1
 - geos=3.6.2
-- gdal=2.2.4
-- geopandas=0.3.0
+- geopandas
 - hdf4=4.2.12=vc9_0
 - hdf5==1.10.1
 - icu=58.1=vc9_1
 - jpeg=9b=vc9_0
 - kealib>=1.4.7
 - krb5=1.14.2=vc9_0
-- libgdal=2.2.4
 - libiconv=1.14=vc9_4
 - libnetcdf>=4.4.1.1
 - libpng=1.6.34
 - libpq=9.6.3=vc9_0
 - libpython=2.0
 - libspatialindex=1.8.5=vc9_1
 - libspatialite
+- libssh2=1.8.0=h0990ea7_2
 - libtiff
 - libxml2=2.9.4=vc9_4
-- matplotlib=2.0.2=py27_2
 - munch=2.2.0=py27_0
 - openjpeg=2.3.0
 - openpyxl
-- openssl=1.0.2l=vc9_0
-- pandas=0.20.3=py27_1
```

Comment:
@daren-thomas I prefer we install the exact version of every library to avoid compatibility issues. 

If we leave it open, a new library we install will update the version of one of these libraries without us realizing and then break compatibility with another library. is that right? maybe I am missing something.

Reply:
yes and no. the exact version will eventually not be hosted anymore. that is what happened this time. so either way, things will break. that is why we do tests - we install a fresh environment everytime a pull request is made. and we try to exercise as much code as possible. this is why writing these tests are so important and why i'd like more help with that. so while we can fix the versions exactly, we don't really remove the problem. how would you like to do it?